src/main.ts

```diff
@@ -99,6 +167,22 @@ async function main() {
 		return await handler.fetchBalance(hash, address);
 	});
 
+	// GET payout information for an address.
+	//
+	// Paths:
+	// - `address`: The _Stash_ address for staking.
+	// - (Optional) `number`: Block number at which to query. If not provided, queries finalized head.
+	//
+	// Returns:
+	// - `at`: Block number and hash at which the call was made.
+	// - `rewardDestination`: The account to which rewards will be paid. Can be 'Staked' (Stash
+	//   account, adding to the amount at stake), 'Stash' (Stash address, not adding to the amount at
+	//   stake), or 'Controller' (Controller address).
+	// - `bonded`: Controller address for the given Stash.
```

Comment:
Should this mention the controller address can also possibly be the stash address? I think that can get a little confusing.

Reply:
Maybe I'll just link to the ref doc of the Staking pallet. But I think this should just say what this function does, not try to explain all of substrate.